applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -718,6 +711,19 @@ int unicast_server_enable(le_audio_receive_cb recv_cb)
 
 	channel_assignment_get(&channel);
 
+	if (!IS_ENABLED(CONFIG_BT_CSIP_SET_MEMBER_TEST_SAMPLE_DATA)) {
+		if (!strcmp(CONFIG_BT_SET_IDENTITY_RESOLVING_KEY_DEFAULT,
+			    CONFIG_BT_SET_IDENTITY_RESOLVING_KEY)) {
```

Comment:
Might be easier to read if it was:
```suggestion
		if (strcmp(CONFIG_BT_SET_IDENTITY_RESOLVING_KEY_DEFAULT,
			    CONFIG_BT_SET_IDENTITY_RESOLVING_KEY) == 0) {
```

Reply:
Yes that looks better.